backend/src/game/cards/Card_JSON_Parser.py

```diff
@@ -0,0 +1,56 @@
+import json
+from enum import Enum
+from typing import List
+
+with open('backend\\src\\game\\cards\\KOTCards.json', "r") as json_file:
+    card_file = json.load(json_file)
+
+
+class CardType(Enum):
```

Comment:
Please extract this enum out into a separate class file.

Reply:
Extracted to CardType.py